vite-base.config.ts

```diff
@@ -12,7 +12,18 @@ import { deferScripsPlugin } from './vite-config/plugins/deferScriptsPlugin'
 import { preloadCSSPlugin } from './vite-config/plugins/preloadCSSPlugin'
 import { excludeBip39Wordlists } from './vite-config/rollup/excludeBip39Wordlists'
 
+import dotenv from 'dotenv';
+// Загрузка переменных из .env
+dotenv.config();
+
+let additional_asset_address = '/';
+
+if (process.env.GITHUB_REPO_NAME) {
+  additional_asset_address = `/${process.env.GITHUB_REPO_NAME}/`;
+}
```

Comment:
Why do you import `dotenv` here if you can just use `process.env.GITHUB_REPO_NAME`?

Reply:
Simplify the code, e.g.:

```js
const publicBasePath = process.env.GITHUB_REPO_NAME ?? "/"
```

Reply:
Now using a singe ENV https://github.com/Adamant-im/adamant-im/pull/745/commits